src/redux/actions/signupAction.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+/* eslint-disable import/prefer-default-export */
+import axios from 'axios';
+import { SIGNUP_ERROR, SIGNUP_SUCCESS } from './actionsTypes';
+
+
+export const signupAction = (user) => async (dispatch) => {
+  try {
+    const res = await axios({
+      method: 'post',
+      url: 'https://devrepublic-bn-backend.herokuapp.com/api/v1/auth/register',
```

Comment:
Is this URI still valid since you changed Orgs?

Reply:
Yes